static/js/jira-connect-ghe.js

```diff
@@ -0,0 +1,62 @@
+const ALLOWED_PROTOCOLS = ["http:", "https:"];
+const ALLOWED_DOMAIN_EXTENSIONS = [".com", ".net"];
```

Comment:
there could be `.org`. Not sure we really need to restrict our top level domain?

Reply:
This has been removed